.github/workflows/ci.yml

```diff
@@ -122,8 +122,8 @@ jobs:
             args: -p fuel-core-client --no-default-features
           - command: test
             args: -p fuel-core-chain-config --no-default-features
-          - command: test
-            args: --manifest-path version-compatibility/Cargo.toml --workspace
+          - command: check
+            args: --manifest-path version-compatibility/Cargo.toml --workspace && cargo test --manifest-path version-compatibility/Cargo.toml --workspace
```

Comment:
Curious why you stuck this into a single command? Feels weird calling `cargo` in part of the `args`.